hack/make-rules/test-e2e-node.sh

```diff
@@ -132,13 +132,15 @@ if [ "${remote}" = true ] && [ "${remote_mode}" = gce ] ; then
 
   # Get the compute zone
   zone=${ZONE:-"$(gcloud info --format='value(config.properties.compute.zone.value)')"}
+  zone=${zone// /}
```

Comment:
so you set it to a space? Or space somehow was set by some automation? I am trying to understand how one can enter this state?

Reply:
the space was set by cli  : 
---
the issue appears when we use gcloud binary installed using scoop (powershell / windows ) and run it with WSL :

# which gcloud
/mnt/c/Users/XXXX/scoop/shims/gcloud
```bash 
# cat /mnt/c/Users/XXXXscoop/shims/gcloud
#!/bin/sh
# C:\Users\XXXX\scoop\apps\gcloud\current\bin\gcloud.cmd
MSYS2_ARG_CONV_EXCL=/C cmd.exe /C "C:\Users\XXXX\scoop\apps\gcloud\current\bin\gcloud.cmd"  "$@" 
The PR will not impact the existing setup but it will handle this case (local running development case).
---

Reply:
thx!

I lgtm-ed, but not an approver here.

@dims are you an approver here? This is simple quality of life change

Reply:
happy to!